### ISSUE TITLE
Lithops: After ion images are migrated, ensure dev environment works without image server proxy

### ISSUE THREAD

**Issue by LachlanStuart** [metaspace2020/metaspace#772]
**Depends on #697**

Currently Colocalization and Ion Thumbnails require an HTTP connection from IBM Cloud to the image storage. Possibly can be fixed with localhost runner - needs to be tested

**Comment by LachlanStuart** [comment]
Closing this - I haven't really had time to check whether this issue exists, but there haven't been any complaints of these failing so it's either working, or not needed for local development.
